cmd/src/gateway_benchmark.go

```diff
@@ -133,15 +159,15 @@ type endpointResult struct {
 	successful int
 }
 
-func benchmarkEndpoint(client *http.Client, url string) time.Duration {
+func benchmarkEndpointHTTP(client *http.Client, url string) time.Duration {
 	start := time.Now()
 	resp, err := client.Get(url)
 	if err != nil {
 		fmt.Printf("Error calling %s: %v\n", url, err)
 		return 0
 	}
-	defer func(Body io.ReadCloser) {
-		err := Body.Close()
+	defer func(body io.ReadCloser) {
+		err := body.Close()
```

Comment:
Ah, thanks! This was IntelliJ's autogenerated code. Will watch out for this in the future.